pkgs/applications/science/biology/whisper/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, fetchurl }:
+
+stdenv.mkDerivation {
+  pname = "whisper";
+  version = "2.0";
+
+  src = fetchFromGitHub {
+    owner = "refresh-bio";
+    repo = "Whisper";
+    rev = "v2.0";
+    sha256 = "100p2pqhli123wnqkxrvjcwnmlvcp1rk63whh121jannnw81rh2m";
+  };
+
+  patches = [ (fetchurl {
+    url = "https://github.com/refresh-bio/Whisper/commit/9ad77e7de68d91d9427cfbe6211e83cea89206ab.patch";
```

Comment:
Can we ask repository owners to bump a minor version instead?

Reply:
Can always ask! I'll open an issue.

Reply:
Expression updated and tested with new release.